build/run.js

```diff
@@ -368,21 +393,25 @@ async function processPackageConfigs() {
 // === Main ===
 // ============
 
-async function updateBuildVersion () {
+async function updateBuildVersion (target) {
     let config        = {}
     let configPath    = paths.dist.buildInfo
     let exists        = fss.existsSync(configPath)
     if(exists) {
         let configFile = await fs.readFile(configPath)
         config         = JSON.parse(configFile)
     }
-    let commitHashCmd = await cmd.run_read('git',['rev-parse','--short','HEAD'])
-    let commitHash    = commitHashCmd.trim()
-    if (config.buildVersion != commitHash) {
```

Comment:
why this was removed? We should not write config file when not needed, as it may cause unneeded recompilation of JS

Reply:
We need to update it always because the target OS can change now. This is no  longer only dependent on the commit has but multiple parameters. Should we actually verify that we only rewrite the file if one of them has changed?

Reply:
Yep, lets compare it